src/allencell_ml_segmenter/main/viewer.py

```diff
@@ -60,8 +61,8 @@ def get_all_shapes(self) -> list[ShapesLayer]:
             if isinstance(l, Shapes)
         ]
 
-    def add_labels(self, data: np.ndarray, name: str) -> None:
-        self.viewer.add_labels(data, name=name)
+    def add_labels(self, data: np.ndarray, **kwargs: Any) -> None:
+        self.viewer.add_labels(data, **kwargs)
```

Comment:
Is this how you name a layer / labels?

Do we actually need Napari level names if we are implementing our own dict in the data model for managing layers?

Reply:
It seems like by switching to `**kwargs`, we are thinning the separation between our `Viewer` interface and `napari.Viewer`. I think if this is the route we're going down, we should at minimum note the 'expected' `**kwargs` in a comment. (something like 'Commonly used kwargs are x and y, see napari.viewer.add_labels documentation for a list of all kwargs possible')